app/model/family.py

```diff
@@ -26,10 +26,23 @@ class FamilyReadDTO(BaseModel):
 class FamilyWriteDTO(BaseModel):
     """A JSON representation of a family for writing."""
 
-    import_id: str
+    # import_id: not included as this is in the request path
```

Comment:
If we're keeping this commented out code for reference, it might be better off as text in the docstring?  Happy to chat about this.

Reply:
Yeah good point ... https://linear.app/climate-policy-radar/issue/PDCT-493/tidy-up-doc-strings-to-produce-documentation